packages/hydrooj/locales/ko_KR.yaml

```diff
@@ -0,0 +1,821 @@
+__langname: 한국어
+'"{data}" copied to clipboard!': “{data}”복사 완료
+'{} is a moderator of this domain.': '{}가 그룹의 관리자입니다.'
+'{0} comments': '{0} 댓글'
+'{0} discussions': '{0} 토론'
+'{0} limit exceeded (limit: {2} operations in {1} seconds).': '{0} 제한 초과：{1} 초 {2} 번'
+'{0} problems': '{0} 문제'
+'{0} sections': '{0} 단락'
+'{0} solutions': '{0} 해설'
+'{0} views': '{0} 조회수'
+'{0}: Permissions': '{0}: 권한'
+'{0}: Roles': '{0}: 역할'
+'{0}: Users': '{0}: 사용자'
+'# My Submissions': '내 제출 기록'
+'# Submissions': '제출 기록'
+'↓ # Comments': '↓ 답글'
+'↓ # Enrollees': '↓ 등록'
+'↓ # Likes': '↓ 좋아요'
+'↓ # Stars': '↓ 북마크'
+'↓ Create Time': '↓ 생성 시각'
+'↓ Update Time': '↓ 업데이트 시각'
+'A domain ID cannot be changed after creation. It is supposed to be:': '그룹 ID는 수정할 수 없으며 다음과 같아야 합니다.：'
+'Copy "{data}" failed :(': '복사“{data}”실패 :('
+'Copy failed :(': '복사 실패 :('
+'Current dataset: {0}': '현재 데이터셋: {0}'
+'Discussion: Terms Of Service': 토론 약관
+'Format: category 1, category 2, ..., category n': 형식：분류1, 분류2, ..., 분류n
+'Hello, {0}! You can click following link to active your new email of your Hydro account:': 안녕하세요!，{0}！다음 링크를 눌러 이메일을 인증하세요：
+'Hello, {0}! You can click following link to reset the password of your Hydro account:': 안녕하세요，{0}！다음 링크를 눌러 비밀번호를 초기화하세요.：
+'Hello! You can click following link to sign up your Hydro account:': 다음 링크를 눌러 로그인하십시오. ：
+'Join {0}': '참여 {0}'
+'No': 아니오
+'Note: Problem title may not be hidden.': '주의: 문제 제목은 숨길 수 없습니다.'
+'Role {1} already exists in domain {0}.': '역할 {1}이(가) 이미 {0}에 존재합니다.'
+'Solved {0} problems, RP: {1} (No. {2})': '총 {0} 문제 해결.RP: {1} (No. {2})'
+'The value `{1}` of {0} already exists.': '{0} 의 값 `{1}` 이 이미 존재합니다.'
+'We use <a href="https://en.gravatar.com/" target="_blank">Gravatar</a> to present your avatar icon.': <a href="https://cn.gravatar.com/" target="_blank">Gravatar</a>를 만들어 보세요!'
```

Comment:
why "!"

Reply:
Just suggesting to use gravatars. Removed it.